stable/distribution/values.yaml

```diff
@@ -74,9 +74,9 @@ mongodb:
   mongodbExtraFlags:
   - "--wiredTigerCacheSizeGB=1"
   mongodbRootPassword:
+  mongodbDatabase: bintray
```

Comment:
mongo database name should be 'distribution'

Reply:
@liorhson see https://github.com/jfrog/charts/pull/27